repose-aggregator/components/filters/keystone-v2-filter/src/main/resources/META-INF/schema/config/keystone-v2-common.xsd

```diff
@@ -100,7 +103,9 @@
                     </xs:documentation>
                 </xs:annotation>
                 <xs:complexType>
+                    <!-- TODO: Does the authorization filter need a default-tenant-quality and roles-tenant-quality? -->
```

Comment:
It seems to me like those two attributes should not mean anything in the authorization filter. Currently they do nothing.

This is kind of painful to tease out, but I think we could do it. It's easy enough to remove these from this type and extend this type in the authentication filter, but then to require the new type means we probably have to restrict the root config type. So then we end up with a restriction and extension of the root config type in order to straighten things out.

Reply:
I think originally the thinking was that authorization was going to long term be writer of these headers so the configuration made sense here, to end up in both. We've teased out a little more how we are actually going to implement, so it makes less sense now.